sql/database.sql

```diff
@@ -9949,3 +9949,11 @@ CREATE TABLE `form_therapy_groups_attendance` (
   activity	tinyint(4),
   PRIMARY KEY (`id`)
 ) ENGINE=InnoDB ;
+
+DROP TABLE IF EXISTS `patient_birthday_alert`;
+CREATE TABLE `patient_birthday_alert` (
+  `pid` bigint(20) NOT NULL,
+  `user_id` bigint(20) NOT NULL,
```

Comment:
for me:
likely need to default these to 0 if not allow NULL (it should never happen, but then at least predictable if it does)